# Website Conversion Rate
A company is testing two versions of a landing page (A and B) to see which one has a higher conversion rate.

Dataset (Generated Sample)
Group A (old design): 500 visitors, 50 conversions
Group B (new design): 480 visitors, 60 conversions
Hypothesis:

Null (H0): No difference in conversion rates between A and B.

Alternative (H1): Conversion rates are different.

In [7]:
import numpy as np
from statsmodels.stats.proportion import proportions_ztest

In [8]:
# Data
conversions = np.array([50, 60])  # Successes
visitors = np.array([500, 480])   # Trials

In [10]:
# Proportion test
z_score, p_value = proportions_ztest(conversions, visitors, alternative='two-sided')

In [11]:
# Print the Z-score and P-value
print(f'Z-score: {z_score:.4f}')
print(f'P-value: {p_value:.4f}')

Z-score: -1.2394
P-value: 0.2152


The P-value (0.2152) is greater than 0.05, so we fail to reject the null hypothesis. There is no significant difference in conversion rates between Group A and Group B.